docs/gettingstarted/structuring.md

```diff
@@ -1,54 +1,83 @@
+If you do not own a domain, **do *NOT* use it for your top level package!**  
+It is perfectly acceptable to start your package with anything, such as your name/nickname, or the name of the mod. For example, `username.mods` would be a good top level package. 
 
-After the top level package (if you have one) you append a unique name for your mod, such as `examplemod`. In our case it will end up as `com.example.examplemod`.
+After you pick a top level package, you should append a unique name to it for your mod. Some good examples would be `com.examplemod.awesomesauce` or `username.mods.awesomesauce`, where **AwesomeSauce** is the name of our mod.
 
 The Mod File
 ------------
 
-Generally, we'll start with a file named after your mod, and put into your package. This is the *entry point* to your mod
-and will contain some special indicators marking it as such.
+Generally, we'll start with a file named after your mod, and put into your package. This is the *entry point* to your mod and will contain some special annotations marking it as such. These annotations are `@Mod`, `@Instance`, and `@EventHandler`. Let's take a look at this example:
+
+    package com.examplemod.awesomesauce;
+        
+    import net.minecraft.init.Blocks;
+    import net.minecraftforge.fml.common.Mod;
+    import net.minecraftforge.fml.common.Mod.EventHandler;
+    import net.minecraftforge.fml.common.event.FMLInitializationEvent;
+        
+    @Mod(modid = "awesomesauce", name = "AwesomeSauce", version = "0.1")
+    public class AwesomeSauceMod
+    {
+        @Instance
+        public static AwesomeSauceMod instance = new AwesomeSauceMod();
+        
+        @EventHandler
+        public void init(FMLInitializationEvent event)
+        {
+            // some example code
+            System.out.println("DIRT BLOCK >> "+Blocks.DIRT.getUnlocalizedName());
+        }
+    }
 
 What is `@Mod`?
 -------------
 
-This is an annotation indicating to the Forge Mod Loader that the class is a Mod entry point. It contains various metadata about the mod. It also designates the class that will receive `@EventHandler` events. More information can be found at... (Coming Soon)
+`@Mod` is the annotation indicating that the class is a *Mod entry point* for Forge Mod Loader. It contains various bits of Metadata that the mod will use as well as modifiers that tell Forge how to handle your mod.
 
-You can find an example mod in the [Forge src download](http://files.minecraftforge.net/).
+Now, the `@Mod` annotation can take quite a few arguments, but the only three *required* ones are `modid`, `name`, and `version`.
+* `modid` is the unique identifier and *internal* name of the mod, and takes a `String`. Usually, this is the same name as the mod, but in all lowercase. It's used internally by Forge and by other Mod Developers that wish to find your mod through Forge (for instance, to use your APIs). 
```

Comment:
You should note the new requirements here. (It must be all lowercase and not more than 64 characters long)